client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -0,0 +1,320 @@
+import useStyles from './useStyles';
+import Typography from '@material-ui/core/Typography';
+import { FormLabel, OutlinedInput, Select, MenuItem, TextField, Button, Box, Menu, Switch } from '@material-ui/core';
+import React, { useState } from 'react';
+
+interface UserData {
```

Comment:
```suggestion
interface UserProfile {
```
Won't UserProfile be a more suitable name for this interface? :)